db-pull-on-boot/templates/db-pull.service.j2

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=DB Pull Service
+
+[Service]
+type=oneshot
+ExecStart=/home/ubuntu/db-push.sh
```

Comment:
```suggestion
ExecStart=dest: {{ chia_root }}/db-pull.sh
```

Since we use chia_root var when copying the file in to place, we should also use that in this template to make sure they always match if set to a non default value